### PR TITLE
Feature/nature page

### PR DESCRIPTION
**Pull request by dev129**
This piece of code contains the information about Nature Page, the Read More links are not activated due to lack of information, the responsiveness test is working well, the image can be changed as per the user request. The nature page can be found in src/app/nature/page.tsx and the images are present in public folder.

### CODE REVIEW

**Comment by dev129** [comment]
Hi @ajaynegi45 , please check this and let me know any other changes you need to see.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "bcryptjs": "^2.4.3",
     "drizzle-kit": "^0.24.2",
     "drizzle-orm": "^0.33.0",
-    "next": "14.2.5",
+    "next": "^14.2.15",
```
Comment (ajaynegi45) [inline package.json:19]: Don't change the version of next

**Review by ajaynegi45 — APPROVED** [review]
Hi @ ,

Thank you for your prompt work on this issue. I appreciate your dedication to the project.

I have reviewed your work, and I need some changes.

- Design is not up to the mark

Once again, thank you for your contribution! ❤️

File: package.json (hunk 1 above)
Comment (dev129) [reply package.json:19]: Hi @ajaynegi45, 
Regarding the design, I’ve strictly followed the Figma file provided, ensuring everything matched as closely as possible. If there were any enhancements needed, let me know. I also believe you had a chance to review the design from the video I shared, where I walked through the page.

**Pull request closed without merging**